src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -152,7 +155,7 @@ public void showAccountDeleted(String name) {
     }
 
     public void showNoAccountFound() {
```

Comment:
Naming sense is problematic. ShowNoAccountFounfld implies that there are no accounts in the savefile at all. showAccountNotFound() or showUserNameNotFound() might be bettet